data_structures/binary_tree/basic_binary_tree.py

```diff
@@ -26,23 +26,27 @@ def display(tree: Node) -> None:  # In Order traversal of the tree
     return
 
 
-def depth_of_tree(
-    tree: Node,
-) -> int:  # This is the recursive function to find the depth of binary tree.
+def depth_of_tree(tree: Node,) -> int:
+    """
+    Recursive function that finds the depth of a binary tree.
+    """
+
     if tree is None:
         return 0
     else:
```

Comment:
```suggestion
    return 1 + max(depth_l_tree, depth_r_tree) if tree else 0
```
...and then delete the rest of this function.

Reply:
did this, thanks!